library/Zend/View/Helper/PartialLoop.php

```diff
@@ -74,4 +94,51 @@ public function getPartialCounter()
     {
         return $this->partialCounter;
     }
+
+    /**
+     * Set object key in this loop and any child loop
+     *
+     * @param string $key
+     */
+    public function setObjectKey($key)
```

Comment:
Guess my documentation skills needs to be improved then :) Pushed a fix for it now.